src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterTest.java

```diff
@@ -29,7 +29,7 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Please import only the necessary things instead of complete file

Reply:
done